worlds/sc2/item_descriptions.py

```diff
@@ -515,7 +518,10 @@ def _ability_desc(unit_name_plural: str, ability_name: str, ability_description:
     item_names.STARTING_SUPPLY: "Increases the starting supply for all missions.",
     item_names.NOTHING: "Does nothing. Used to remove a location from the game.",
     item_names.NOVA_GHOST_VISOR: "Reveals the locations of enemy units in the fog of war around Nova. Can detect cloaked units.",
-    item_names.NOVA_RANGEFINDER_OCULUS: "Increaases Nova's vision range and non-melee weapon attack range by 2. Also increases range of melee weapons by 1.",
+    item_names.NOVA_RANGEFINDER_OCULUS: inspect.cleandoc("""
+        Increases Nova's vision range and non-melee weapon attack range by 2.
+        Also increases range of melee weapons by 1.
+    """),

```

Comment:
I wonder if a more scalable solution to tooltip space is to just do a `.replace('. ', '.\n')` on the description when building the tooltip. So each sentence would get its own line.